images/build/Dockerfile

```diff
@@ -177,22 +176,14 @@ RUN set -ex \
  && ln -s 5.6.0 /opt/npm/5.6 \
  && ln -s 5.6 /opt/npm/5
 
-###
-# Python intermediate stages
-# Docker doesn't support variables in `COPY --from`, so we're using intermediate stages
-###
-FROM mcr.microsoft.com/oryx/base:python-build-2.7-${PYTHON_BASE_TAG} AS py27-build-base
-FROM mcr.microsoft.com/oryx/base:python-build-3.6-${PYTHON_BASE_TAG} AS py36-build-base
-###
-# End Python intermediate stages
-###
-
 FROM main AS python
 # https://github.com/docker-library/python/issues/147
 ENV PYTHONIOENCODING UTF-8
 COPY build/__pythonVersions.sh /tmp/scripts
-COPY --from=py27-build-base /opt /opt
-COPY --from=py36-build-base /opt /opt
+RUN cd /tmp/scripts \
+    && . ./__pythonVersions.sh \
+    && ./installPlatform.sh python $PYTHON27_VERSION \
+    && ./installPlatform.sh python $PYTHON36_VERSION
```

Comment:
python 3.8 and 3.7?

Reply:
That is being installed in the slim image. Check `slim.Dockerfile`